client/src/views/users/GreetUserModal.vue

```diff
@@ -0,0 +1,442 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS -->
+
+<template>
+  <ion-page>
+    <ms-wizard-stepper
+      v-show="pageStep > GreetUserStep.WaitForGuest && pageStep < GreetUserStep.Summary"
+      :current-index="getStepperIndex()"
+      :titles="[
+        'Code admin',
+        'Code invité',
+        'Contact details',
+      ]"
+    />
+    <ion-buttons
+      slot="end"
+      class="closeBtn-container"
+    >
+      <ion-button
+        slot="icon-only"
+        @click="cancelModal()"
+        class="closeBtn"
+      >
+        <ion-icon
+          :icon="close"
+          size="large"
+          class="closeBtn__icon"
+        />
+      </ion-button>
+    </ion-buttons>
+    <div
+      class="modal"
+    >
+      <ion-header class="modal-header">
+        <ion-title
+          class="modal-header__title title-h2"
+        >
+          {{ getTitleAndSubtitle()[0] }}
+        </ion-title>
+        <ion-text
+          class="modal-header__text body"
+        >
+          {{ getTitleAndSubtitle()[1] }}
+        </ion-text>
+      </ion-header>
+      <div class="modal-content inner-content">
+        <div
+          v-show="pageStep === GreetUserStep.WaitForGuest"
+          class="step"
+        >
+          <ms-informative-text
+            :icon="caretForward"
+            :text="$t('UsersPage.greet.subtitles.waitForGuest1')"
+          />
+          <ms-informative-text
+            :icon="caretForward"
```

Comment:
The caretForward seems to be the icon set in most ms-informative-text use cases: should we (in a separate issue), set it as default?

Reply:
Yes good idea, there are only a few cases where the icon changes

Reply:
Issue #4986